lib/google-apis/auth-client.js

```diff
@@ -270,11 +358,19 @@ AuthClient.prototype.sendError = function(err, callback) {
 
   if (!this._shouldReportErrors) {
 
-    return;
+    callback(
+        new Error([
+          'Client has not been configured to send errors, please check the',
```

Comment:
'Client' would not be descriptive enough to the end user. However, this can be addressed by moving to the diagnostics common library.